modules/aws-outposts-load-balancer-clb-fix-subnet.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-load-balancer-clb-fix-subnet_{context}"]
+= Ensuring AWS Classic Load Balancers use the correct subnet
+
+When you extend an AWS VPC cluster to an Outpost, automatic discovery by the cloud controller manager might cause AWS Classic Load Balancers to erroneously be added to the Outpost availability zone.
```

Comment:
I think this could be made more clear in any case. Perhaps:
```
When you extend an AWS VPC cluster to an Outpost, the subnet discovery by the cloud controller manager might cause new AWS Classic Load Balancers to be added to the Outpost availability zone.
Because AWS Outposts does not support Classic Load Balancers, you must prevent Classic Load Balancers from being added to the Outpost availability zone by adding an annotation with the correct subnet when you configure them.
```